src/librbd/image/OpenRequest.h

```diff
@@ -38,7 +38,7 @@ class OpenRequest {
    *    \-----> V2_DETECT_HEADER                    |             .
    *                |                               |             .
    *                v                               |             .
-   *            V2_GET_ID                           |             .
+   *            V2_GET_ID|NAME                      |             .
```

Comment:
To me, this implies that the _GET_ID state is mutually exclusive with the _GET_NAME state.